webhooks/service_test.go

```diff
@@ -0,0 +1,159 @@
+package webhooks_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/MainfluxLabs/mainflux/internal/apiutil"
+	"github.com/MainfluxLabs/mainflux/pkg/errors"
+	"github.com/MainfluxLabs/mainflux/pkg/messaging"
+	"github.com/MainfluxLabs/mainflux/pkg/mocks"
+	"github.com/MainfluxLabs/mainflux/webhooks"
+	whMock "github.com/MainfluxLabs/mainflux/webhooks/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	thingID    = "1"
+	token      = "admin@example.com"
+	wrongValue = "wrong-value"
+	emptyValue = ""
+)
+
+func newService() webhooks.Service {
+	things := mocks.NewThingsServiceClient(nil, map[string]string{token: thingID}, nil)
+	webhookRepo := whMock.NewWebhookRepository()
+
+	return webhooks.New(things, webhookRepo)
+}
+
+func TestCreateWebhooks(t *testing.T) {
+	svc := newService()
+
+	cases := []struct {
+		desc     string
+		webhooks []webhooks.Webhook
+		token    string
+		err      error
+	}{
+		{
+			desc:     "create new webhooks",
+			webhooks: []webhooks.Webhook{{ThingID: "1", Name: "test1", Format: "json", Url: "http://test1.com"}, {ThingID: "1", Name: "test2", Format: "json", Url: "https://test2.com"}},
```

Comment:
Create variables for this value and all others. Similar to what is done in the endpoint_tests.go